web/src/components/Badge.tsx

```diff
@@ -1,6 +1,8 @@
+export type PillTypes = 'default' | 'warning';
```

Comment:
Small thing but would it be more correct to call it `PillType`?

Reply:
I'm actually not sure what the convention is but I was thinking that the property `pillType` should be a type of `PillTypes` if that makes sense?

I'd be happy to change it if you wish though.

Reply:
I did a little google search and it seems that the convention is singular, so `PillType` ([Types in typescript](https://imgeorgemor.medium.com/you-dont-know-how-to-write-types-in-typescript-d8cfaf4857e5))

Reply:
Great!

I'll change it then!